lotus-soup/testkit/role_miner.go

```diff
@@ -432,7 +462,24 @@ func (m *LotusMiner) RunDefault() error {
 	return nil
 }
 
-func startStorageMinerAPIServer(t *TestEnvironment, repo *repo.MemRepo, minerApi api.StorageMiner) error {
+func (m *LotusMiner) Halt() {
+	m.t.RecordMessage("halting miner")
```

Comment:
I understand this as if we are stopping the whole miner node, whereas it is just stopping the block producing part of it. I think it makes sense being explicit about it.